pipeline/minimmit/quint/replica.qnt

```diff
@@ -694,23 +694,15 @@ module replica {
     // Invariants
     // -------------------------------------------------
 
-    // Assumptions and thresholds:
-
+    // Assumptions and thresholds.
     val assumptions_valid = all {
-        2*L <= (N - F) + 1, // # Liveness guard, prevents equivocation stall
         CORRECT.size() + FAULTY.size() == N,
         N >= 5*F+1,
-        L + Q > N + F,
-        
-        2*F + 1 <= L,
-        L <= N - 3*F,
-
-        4*F + 1 <= Q,
-        Q <= N - F,
```

Comment:
I think you need to keep this invariant, right? Otherwise `Q=N` could be valid, which would prevent finalization from occurring? Or is `Q = N - F` and `F = 2F+1` hardcoded somewhere?